Smartway.UiComponent/Behaviors/GroupedItemsBehavior.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using Smartway.UiComponent.Inputs;
+using Xamarin.Forms;
+
+namespace Smartway.UiComponent.Behaviors
+{
+    public class GroupedItemsBehavior : Behavior
+    {
+        private static readonly Dictionary<string, List<ISelectableInput>> GroupedItems
+            = new Dictionary<string, List<ISelectableInput>>();
+
+        public static readonly BindableProperty GroupNameProperty = BindableProperty.CreateAttached(nameof(GroupName),
+            typeof(string), typeof(GroupedItemsBehavior), string.Empty, propertyChanged: GroupNameChanged);
+
+        private static void GroupNameChanged(BindableObject bindable, object oldvalue, object newvalue)
+        {
+            if (!(newvalue is string groupName))
```

Comment:
il sort d'où le groupName ? Les noms sont pas super clair

Reply:
pas compris ^^

Reply:
le if c'est pour dire :
if(!newValue is string)
    var groupName = (string) newValue
    
 C'est concaténé :)

Reply:
Ah ok merci je ne connaissais pas.